pom.xml

```diff
@@ -0,0 +1,671 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+  <!-- Model Information -->
+  <modelVersion>4.0.0</modelVersion>
+
+  <!-- Artifact Information  -->
+  <groupId>xyz.block</groupId>
+  <artifactId>web5-parent</artifactId>
```

Comment:
Question: By having the `web5-parent` here, are we publishing any `web5-parent` package? Or it's just used for gluing the subpackages as I've seen references to parent across the board?

Reply:
Yep, we publish the `web5-parent`; it'll be used as the parent of `tbdex-parent`, thereby slimming down all the plugin configs, versioning info, etc necessary at the tbDEX level. Like [this](https://github.com/TBD54566975/tbdex-kt/blob/issue-204/maven-build/pom.xml#L13).